### ISSUE TITLE
ErrorException Undefined index id

### ISSUE THREAD

**Issue by okpierre** [pixelfed/pixelfed#2357]
```

App\Jobs\InboxPipeline\InboxValidator

ErrorException: Undefined index: id in /pixelfed/app/Util/ActivityPub/Helpers.php:293

#0 /pixelfed/app/Util/ActivityPub/Helpers.php(293): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 /pixelfed/app/Util/ActivityPub/Inbox.php(149): App\Util\ActivityPub\Helpers::statusFirstOrFetch()
#2 /pixelfed/app/Util/ActivityPub/Inbox.php(131): App\Util\ActivityPub\Inbox->handleNoteReply()

```

**Comment by NiklausHofer** [comment]
I am getting the same errors on my instance, too.

**Comment by dansup** [comment]
Fixed in #2665
